packages/icons/src/index.js

```diff
@@ -166,6 +171,7 @@ export { default as rotateLeft } from './library/rotate-left';
 export { default as rotateRight } from './library/rotate-right';
 export { default as rss } from './library/rss';
 export { default as search } from './library/search';
+export { default as security } from './library/security';
```

Comment:
Should this be called "shield" instead? While there are a few icons that are somewhat opinionated in their names, in general I think we should try and be _descriptive_ instead — use "paintbrush" instead of "design", etc.

Reply:
Great point, changed in https://github.com/WordPress/gutenberg/pull/32371/commits/2822acc5b75113b6e0b6290a258a24dba7549aa7.